packages/chain-helpers/src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -24,8 +26,12 @@ export const CUSTOM_TYPES: RegistryTypes = {
   BlockNumber: 'u64',
   Index: 'u64',
   RefCount: 'u32',
-
-  ErrorCode: 'u16',
+  AccountInfo: {
```

Comment:
Not sure whether this is required, probably have to remove this again. This is the current `AccountInfo` type from Polkadot.